src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupItem.md

```diff
@@ -18,24 +18,24 @@ Gets the items from a container in Backup.
 ```
 Get-AzRecoveryServicesBackupItem [-Container] <ContainerBase> [[-Name] <String>]
  [[-ProtectionStatus] <ItemProtectionStatus>] [[-ProtectionState] <ItemProtectionState>]
- [-WorkloadType] <WorkloadType> [[-DeleteState] <ItemDeleteState>] [-VaultId <String>]
- [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+ [-WorkloadType] <WorkloadType> [[-DeleteState] <ItemDeleteState>] [[-FriendlyName] <String>]
+ [-VaultId <String>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
```

Comment:
Do we want to add an example of the new usage here?

Reply:
Added